changes/527.changed

```diff
@@ -0,0 +1,2 @@
+Changed Python versions in test suite.
```

Comment:
Nit: This filename should _probably_ be `.housekeeping` since it's not related to anything that the end users would care about.